### PR TITLE
Update project to work with mod_wsgi

### PR DESCRIPTION
**Pull request by ggiamarchi**
Related to https://github.com/RuggedPOD/ruggedpod-web/pull/14

### CODE REVIEW

File: ruggedpod_api/__init__.py [hunk 1]
```diff
@@ -14,3 +14,16 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os.path
+
+from ruggedpod_api.common.conf import YmlConf
+
+CONF_FILE = "/etc/ruggedpod/api-conf.yaml"
```
Comment (migibert) [inline ruggedpod_api/__init__.py:22]: This should be documented in README

File: ruggedpod_api/service_auth.py [hunk 2]
```diff
@@ -47,18 +51,31 @@ class Cypher(object):
...
+    def encrypt(self, raw):
+        iv = Random.new().read(AES.block_size)
+        cipher = AES.new(self.key, AES.MODE_CBC, iv)
+        return base64.b64encode(iv + cipher.encrypt(self._pad(raw)))
 
-    def decrypt(self, cipher):
+    def decrypt(self, enc):
         try:
-            return self.cipher_suite.decrypt(cipher.encode("ascii"))
+            enc = base64.b64decode(enc)
```
Comment (migibert) [inline ruggedpod_api/service_auth.py]: I would not use the parameter to represent both encrypted base64 encoded string and encrypted base64 decoded string.
Maybe you should use two variables "encoded" and "encrypted" ?

File: ruggedpod_api/__init__.py (hunk 1 above)
Comment (ggiamarchi) [inline ruggedpod_api/__init__.py:22]: There's currently no documentation in the README. By the way, the legacy behaviour is still working.

File: ruggedpod_api/service_auth.py (hunk 2 above)
Comment (ggiamarchi) [inline ruggedpod_api/service_auth.py]: Done

**Comment by migibert** [comment]
LGTM

**Pull request merged**